CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1 @@
+- Fix: typo in globals.cpp file (#4621)
```

Comment:
```suggestion
```

The typo is minor and without functional impact in the CB, thus no need of mentioning it at Changelog.

Reply:
Fixed in [47d2c78](https://github.com/telefonicaid/fiware-orion/pull/4622/commits/47d2c785ae75721f69c2aebf16051ebbc40d830a)